# SLOW5 Publications

## Format and toolkit

S/BLOW5 file format, slow5lib (C library) and pyslow5 (Python Library)
> Gamaarachchi, H., Samarakoon, H., Jenner, S.P. et al. Fast nanopore sequencing data analysis with SLOW5. Nat Biotechnol 40, 1026-1029 (2022). [https://doi.org/10.1038/s41587-021-01147-4](https://doi.org/10.1038/s41587-021-01147-4)

slow5tools toolkit
> Samarakoon, H., Ferguson, J.M., Jenner, S.P. et al. Flexible and efficient handling of nanopore sequencing signal data with slow5tools. Genome Biol 24, 69 (2023). [https://doi.org/10.1186/s13059-023-02910-3](https://doi.org/10.1186/s13059-023-02910-3)

Benefits of S/BLOW5 file format over ONT's latest POD5 format
> Hasindu Gamaarachchi, Sasha Jenner, Hiruna Samarakoon, James M Ferguson, Ira W Deveson, The enduring advantages of the SLOW5 file format for raw nanopore sequencing data, GigaScience, Volume 14, 2025, giaf118. [https://doi.org/10.1093/gigascience/giaf118](https://doi.org/10.1093/gigascience/giaf118)

## Core applications

Remote S/BLOW5 access using slow5curl
> Bonson Wong, James M Ferguson, Jessica Y Do, Hasindu Gamaarachchi, Ira W Deveson, Streamlining remote nanopore data access with slow5curl, GigaScience, Volume 13, 2024, giae016, [https://doi.org/10.1093/gigascience/giae016](https://doi.org/10.1093/gigascience/giae016)

The new ex-zd nanopore signal compression
> Jayasooriya K, Jenner SP, Marasinghe P, Senanayake U, Saadat H, Taubman D, Ragel R, Gamaarachchi H, Deveson IW. A new compression strategy to reduce the size of nanopore sequencing data. Genome Research. 35, no. 7 (2025): 1574-1582. [https://doi.org/10.1101/gr.280090.124](https://doi.org/10.1101/gr.280090.124)

Basecalling with S/BLOW5
> Hiruna Samarakoon, James M Ferguson, Hasindu Gamaarachchi, Ira W Deveson, Accelerated nanopore basecalling with SLOW5 data format, Bioinformatics, Volume 39, Issue 6, June 2023, btad352, [https://doi.org/10.1093/bioinformatics/btad352](https://doi.org/10.1093/bioinformatics/btad352)


## Various other applications

Signal simulation
> Gamaarachchi, H., Ferguson, J. M., Samarakoon, H., Liyanage, K., & Deveson, I. W. (2024). Simulation of nanopore sequencing signal data with tunable parameters. Genome Research, 34(5), 778-783.[https://doi.org/10.1101/gr.280090.124](https://doi.org/10.1101/gr.280090.124)

Signal visualisation
> Samarakoon, H., Liyanage, K., Ferguson, J.M., Parameswaran, S., Gamaarachchi, H. and Deveson, I.W., 2024. Interactive visualization of nanopore sequencing signal data with Squigualiser. Bioinformatics, 40(8), p.btae501. [https://doi.org/10.1093/bioinformatics/btae501](https://doi.org/10.1093/bioinformatics/btae501)

Signal alignment
> Gamaarachchi, H., Lam, C.W., Jayatilaka, G. et al. GPU accelerated adaptive banded event alignment for rapid comparative nanopore signal analysis. BMC Bioinformatics 21, 343 (2020). [https://doi.org/10.1186/s12859-020-03697-x](https://doi.org/10.1186/s12859-020-03697-x)

Pore model generation
> Hiruna Samarakoon, Yuk Kei Wan, Sri Parameswaran, Jonathan Göke, Hasindu Gamaarachchi, Ira W Deveson, Leveraging basecaller’s move table to generate a lightweight k-mer model for nanopore sequencing analysis, Bioinformatics, Volume 41, Issue 4, April 2025, btaf111, [https://doi.org/10.1093/bioinformatics/btaf111](https://doi.org/10.1093/bioinformatics/btaf111)


